src/dotty/tools/dotc/ast/Desugar.scala

```diff
@@ -46,7 +46,11 @@ object desugar {
      */
     override def ensureCompletions(implicit ctx: Context) =
```

Comment:
The comment above could be updated to note that we also need to have the class itself completed.